src/components/colorscale/color_attributes.js

```diff
@@ -64,14 +69,16 @@ module.exports = function makeColorScaleAttributes(context) {
         autocolorscale: extendDeep({}, colorScaleAttributes.autocolorscale, {
             description: [
                 'Has an effect only if `', context, '.color` is set to a numerical array.',
-                ' Determines whether or not the colorscale is picked using',
-                ' values inside `', context, '.color`.'
+                ' Determines whether the colorscale is the default palette (`autocolorscale: true`)',
+                ' or the palette determined by `', context, '.colorscale`.'
```

Comment:
It is important to communicate that there are _three_  default palettes. 

The palette selected depend on the whether the data has only positive, only negative or else numbers (see [logic](https://github.com/plotly/plotly.js/blob/32293e36c7e4ee47afd9c033f7dc226ccc8d666e/src/components/colorscale/calc.js#L54-L56)).